x/liquidity/spec/02_state.md

```diff
@@ -1,105 +1,117 @@
-<!--
-order: 2
--->
+<!-- order: 2 -->
 
-# State
+ # State
+
+The Liquidity Module `x/liquidity` keeps track of the Pool and PoolBatch states. The state represents your app at a given moment.
```

Comment:
I was a bit confused by the DepositMsgState, WithdrawMsgState, and SwapMsgState states (are these separate states or are they considered to be part of the PoolBatch states? It seems PoolBatch, DepositMsgState, WithdrawMsgState, and SwapMsgState share the same parameters, but they each have a different structure. Can you explain? Do we need to be more clear in the doc?

Reply:
these are separate states for each Msg, 
common variables can be refactored into common structures, but each declared to avoid nested structures.